### PR TITLE
Compelted javascript-primitiaves-variables

### PR DESCRIPTION
**Pull request by Eugenepark215**
Screen Shot 2022-08-29 at 9 14 22 AM

### CODE REVIEW

File: javascript-primitives-and-variables/main.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+fullName = 'Eugene Park';
+isCool = true;
+totalPets = 1;
+
+console.log(fullName);
+console.log(isCool);
+console.log(totalPets);
+console.log(typeof fullName);
+console.log(typeof isCool);
+console.log(typeof totalPets);
```
Comment (codingcodymiller) [inline javascript-primitives-and-variables/main.js]: image

File: javascript-primitives-and-variables/main.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+var unknown;
+unknown = null;
+console.log(unknown);
+console.log(typeof unknown);
```
Comment (codingcodymiller) [inline javascript-primitives-and-variables/main.js]: image

File: javascript-primitives-and-variables/main.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+var never;
+console.log(never);
+console.log(typeof never);
```
Comment (codingcodymiller) [inline javascript-primitives-and-variables/main.js]: image

File: javascript-primitives-and-variables/main.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+var fullName;
+var isCool;
+var totalPets;
+
+fullName = 'Eugene Park';
+isCool = true;
+totalPets = 1;
```
Comment (codingcodymiller) [inline javascript-primitives-and-variables/main.js:7]: This is not necessary in JavaScript. You can declare a variable and assign a value on the same line.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by Eugenepark215** [comment]
Screen Shot 2022-08-29 at 9 21 48 AM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**